advanced-usage/index.md

```diff
@@ -38,15 +39,15 @@ interested in the basic functionality you can click the button below:
 </div>
 
 Any pages that are extended by (or only available in) Pirate Mode are shown in
-[dark mode](#settings), and described with grey text.
+[dark mode](#display-mode-management), and described with grey text.
```

Comment:
this is weird.

Reply:
Lol, what about it is weird?

Reply:
The dark mode thing. Is that a special tag or something?

Reply:
You know how BlueOS has light and dark display modes? I used dark mode for the screenshots that include pirate-mode-only functionalities. That was already the case for the BlueOS-1.0 docs though - if you click the [hide advanced functionalities](https://docs.bluerobotics.com/ardusub-zola/software/onboard/BlueOS-1.0/advanced-usage/#pirate-mode) button then it only shows light-mode screenshots, with the simplified interface.

Unless you're just talking about the fact I'm using a markdown hyperlink that jumps to a section within the current page?

Reply:
To be clearer: is the text saying that pirate mode functionality is only available in dark mode?

Reply:
Oh I see. I've updated it now, hopefully that helps :-)